.github/workflows/docker.yaml

```diff
@@ -0,0 +1,38 @@
+name: Docker Image CI
```

Comment:
We're using CircleCI, not github actions and only plan to use one CI system/provider. So for this to move forward, it'd need to use CircleCI

Reply:
I'm afraid the easiest way right now is to use GitHub Actions. This is because Docker provides QEMU([docker/setup-qemu-action](https://github.com/docker/setup-qemu-action)) and Buildx([docker/setup-buildx-action](https://github.com/docker/setup-buildx-action)) integration into GitHub Actions.

If we are going to use CircleCI for building multi-arch images, we will need to install [docker/buildx](https://github.com/docker/buildx) manually and setup QEMU.